### DISCUSSION TITLE
Break progress bar animation should happen every half beat

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#30075] [Ideas]
Basically title, the progress bar syncing to the beat is kinda neat, but would probably feel better if it happened every half beat, rather than every beat like it currently is.

**Comment by peppy** [comment]
I tried that and it looked weird.

**Reply by vatei** [reply]
Hmm, this proposal could maybe help then https://github.com/ppy/osu/discussions/30254

**Reply by peppy** [reply]
I also tried that and it didn't seem right. Needs more experimentation for sure.l
